src/main/resources/application.properties

```diff
@@ -7,17 +7,38 @@
 #zeebe.client.cloud.clientSecret=XXX
 
 # or local
-#zeebe.client.broker.gateway-address=127.0.0.1:26500
-#zeebe.client.security.plaintext=true
+#camunda.client.zeebe.base-url=http://localhost:26500
+#camunda.client.zeebe.gateway-url=http://localhost:26500
 
+camunda.client.mode=oidc
+camunda.client.auth.client-id=zeebe
+camunda.client.auth.client-secret=tSpZfhCadVrSXqIpMtdMKveczmqFXnkD
+camunda.client.auth.issuer=https://performance.testing.engineering.telflow.com/auth/realms/telflow
+
+camunda.client.zeebe.enabled=true
+camunda.client.zeebe.prefer-rest-over-grpc=false
+camunda.client.zeebe.base-url=http://localhost:26555
+camunda.client.zeebe.gateway-url=http://localhost:26555
+camunda.client.zeebe.audience=zeebe-api
+
+camunda.client.identity.enabled=true
+camunda.client.operate.enabled=false
+camunda.client.tasklist.enabled=false
+camunda.client.optimize.enabled=false
+
+logging.level.root=info
+# Enable DEBUG logging level for Spring framework
+
+# Enable DEBUG logging level for your project's package
+#logging.level.org.camunda.community.benchmarks=DEBUG
 
 server.port=8088
 management.endpoints.web.exposure.include=*
 
 # Main benchmark configurations
 ######################
 benchmark.startProcesses=true
-benchmark.startPiPerSecond=25
+benchmark.startPiPerSecond=17
```

Comment:
what is this magic number (the biggest that works in our test, something else)?

Reply:
the app will create 17 process instances per second to test, it will crease when the app doesn't get response resources exhausted